charts/cluster-api-cluster-openstack/templates/control-plane.yaml

```diff
@@ -41,6 +41,8 @@ spec:
           {{- include "openstack.taints.control-plane" . | nindent 8 }}
       patches:
         directory: /etc/kubernetes/patches
+    preKubeadmCommands:
+      - bash /tmp/kubeadm/patches/kube-apiserver-patch.sh
```

Comment:
Could we possibly do 'bash myscript -c {{ .Values.api.cpuLimitPercent }} -m {{ .values.api.memoryLimitPercent }}` to appease my desire for not having to fuck about when the shit hits the fan and it needs changing?

You can then use optargs in the script as so...

```bash
#!/bin/bash

CPU_PERCENT="50"
MEM_PERCENT="50"

while getopts "c:m:" arg; do
	case "$arg" in
		c)
			CPU_PERCENT="${OPTARG}"
			;;
		m)
			MEM_PERCENT="${OPTARG}"
			;;
	esac
done

echo "${CPU_PERCENT} ${MEM_PERCENT}"
```

Damn, that was patronizing.  You derive what's best for you 😸

Reply:
Yup I anticipated you might suggest as much, and I've done as you recommend.